lib/server.js

```diff
@@ -86,8 +86,16 @@ FastBootServer.prototype.insertIntoIndexHTML = function(title, body, head) {
 };
 
 FastBootServer.prototype.handleSuccess = function(res, path, result, startTime) {
-  this.log(200, 'OK ' + path, startTime);
-  res.send(this.insertIntoIndexHTML(result.title, result.body, result.head));
+  var contentFor = result.contentFor;
+  var headers = result.headers;
+
+  this.log(result.statusCode, 'OK ' + path, startTime);
```

Comment:
Instead of "OK", how about using the correct text for the given status code?
```
var HttpStatusCodes = require('http-status-codes');
this.log(result.statusCode, HttpStatusCodes.getStatusText(result.statusCode), path, startTime);
```